Jacks27/population-management-system
# [Finishes #16452199] updated requirement

#### What does this PR do?
create table users , locations , sub-locations
#### Description of Task to be completed?
connect the database using postgres and psychopg2
#### How should this be manually tested?
clone the repo cd into it then run pip install requirements .txt 
start posgress sever run the command \c pms the ldt to check if the relations have been created 

#### What are the relevant pivotal tracker stories?
#164521963
#### Screenshots (if appropriate)
#### Questions:
